src/etc/inc/util.inc

```diff
@@ -2474,5 +2488,21 @@ function get_smart_drive_list() {
 	sort($disk_list);
 	return $disk_list;
 }
+/* Write the DHCP6 DUID file */
```

Comment:
Please keep blank line separating 2 functions and no blank lines between comment and function prototype